src/popover/container.tsx

```diff
@@ -34,10 +33,12 @@ export interface PopoverContainerProps {
   variant?: 'annotation';
   // When keepPosition is true, the popover will not recalculate its position when it resizes nor when it receives clicks.
   keepPosition?: boolean;
+  // When allowVerticalScroll is true, we will scroll to the the popover if it overflows the viewport even when choosing the best possible position for it.
+  // Do not use this if the popover is open on hover, in order to avoid unexpected movement.
+  allowVerticalScroll?: boolean;
```

Comment:
A better name would be `allowScrollToFit`, something that suggests there will be scroll outside, not inside the content

Reply:
Changed